test-it/ExpectedResults/UnflattenJSONEmptyAndMismatchdArray.vtr

```diff
@@ -11,9 +11,9 @@ JSON null array regression test
  Err = (true -375008 'Unflatten From JSON in _Functionghtmlvi')
 2D Empty Unflattened JSON |{"array":[[]]}| to mismatched 1D array : (())
  Err = (true -375008 'Unflatten From JSON in _Functionghtmlvi')
-1D Unflattened JSON |{"array":[1,2,3]}| to mismatched 2D array : (())
```

Comment:
That seems to apply to every test in this file, not just this one case.

I'm not sure why every test in this file is operating on clusters that have a single field that is an array instead of operating on arrays directly.

We should probably either:
1. Refactor all the test names to include a description that they are comparing clusters that have arrays or
2. Refactor the test to operate on arrays directly

If you feel strongly that that would be valuable then it can be done in a separate PR. That is refactoring tech debt unrelated to this change.

Reply:
Lets talk about which change is the best to make and it can be done in a separate PR. 👍

Reply:
It can be done in a separate PR. It was confusing when I looked at the nesting parentheses. Please feel free to create a github issue.